Tests/ClientRuntimeTests/NetworkingTests/URLSession/FoundationStreamBridgeTests.swift

```diff
@@ -104,7 +104,7 @@ class FoundationStreamBridgeTests: XCTestCase {
     }
 }
 
-private class TestLogger: LogAgent {
+private class TestLogger: LogAgent, @unchecked Sendable {
```

Comment:
It's preferred to fix any sendability issues when possible rather than just sidestepping the issue with `@unchecked`.

`@unchecked` is typically for when we rely on some mechanism for concurrent safety that the compiler can't check, such as GCD or C code that uses primitive locking.

Reply:
Summary of offline discussion:

The test loggers have string array "storage" that it appends logs to, and that's how the relevant tests validate on expected logger output.making test loggers an actor or putting @MainActor tag to the string array var doesn't work bc it complains about how actor isolated property / function can't count for protocol conformance. And using lock doesn't get recognized by concurrency check.

Keeping these the way they are since storing logs in nested property is something unique for testing purposes.